Foundation/FileManager+Win32.swift

```diff
@@ -358,8 +360,7 @@ extension FileManager {
     }
 
     internal func _copyOrLinkDirectoryHelper(atPath srcPath: String, toPath dstPath: String, variant: String = "Copy", _ body: (String, String, FileAttributeType) throws -> ()) throws {
-        var faAttributes: WIN32_FILE_ATTRIBUTE_DATA = WIN32_FILE_ATTRIBUTE_DATA()
-        do { faAttributes = try windowsFileAttributes(atPath: srcPath) } catch { return }
+        let faAttributes = try windowsFileAttributes(atPath: srcPath)
```

Comment:
How come we don't need the exception to propagate here?

Reply:
Previously, if srcPath didn't exist, it would catch the error from windowsFileAttributes and _copyItem would silently not do anything. This way it bubbles up the exception and throws a file not found as it should.